src/editorview.h

```diff
@@ -72,6 +73,9 @@ Q_OBJECT
 
   GripItem* activeVertex() const;
   MeshItem* activeMesh() const;
+  GripItem* getSnapGrip() const;
```

Comment:
In the Qt coding style guide, accessors are named as a noun (`snapGrip()`), not a verb (`getSnapGrip()`). This is why you see the `m_` prefix on the backing variables in some of my code.

Reply:
Thanks. I knew there was a reason you were doing things that way but I forgot the m_ part. I will fix that

Reply:
(With the exception being booleans, which are `isFoo()` or `areFoo()` paired with `setFoo()`)